dynamometer-infra/src/main/java/com/linkedin/dynamometer/ApplicationMaster.java

```diff
@@ -328,7 +328,7 @@ public Boolean get() {
         amOptions.getDataNodeVirtualCores() + " vcores, ");
     for (int i = 0; i < numTotalDataNodeContainers; ++i) {
       ContainerRequest datanodeAsk = setupContainerAskForRM(amOptions.getDataNodeMemoryMB(),
-          amOptions.getDataNodeVirtualCores(), 1);
+          amOptions.getDataNodeVirtualCores(), 1, amOptions.getDataNodeNodeLabelExpression());
```

Comment:
what's the behavior if this is empty ?

Reply:
The default is `""`, so it uses the default (unlabeled) node label.